### PR TITLE
Add job to build and push image for openshift-powervs/cluster-api-pro…

### PR DESCRIPTION
**Pull request by mkumatag**
…vider-powervs

### CODE REVIEW

File: config/jobs/openshift-powervs/cluster-api-provider-powervs/postsubmit.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    namespace: ${namespace}
+                    user: default-user
+                current-context: default-context
+                users:
+                - name: default-user
+                  user:
+                    token: ${token}
+                EOL
+                export KUBECONFIG=$(pwd)/kubeconfig
+                kubectl build --push --registry-secret quay-powercloud-regcred --namespace image-builder --build-arg VERSION=v0.0.0-master-${commit} -t quay.io/powercloud/origin-powervs-machine-controllers:v0.0.0-master-${commit} -f Dockerfile .
```
Comment (Rajalakshmi-Girish) [inline config/jobs/openshift-powervs/cluster-api-provider-powervs/postsubmit.yaml:50]: Are the two build commands to add two different tags?
I think we can pass two tags in one command like in https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ocp-automation/openshift-install-power/openshift-install-power-images.yaml#L63

**Review by Rajalakshmi-Girish — CHANGES_REQUESTED** [review]

File: config/jobs/openshift-powervs/cluster-api-provider-powervs/postsubmit.yaml (hunk 1 above)
Comment (mkumatag) [reply config/jobs/openshift-powervs/cluster-api-provider-powervs/postsubmit.yaml:50]: yes, thanks for the pointer. I will make that change.

**Comment by mkumatag** [comment]
@Rajalakshmi-Girish addressed the comment, ptal latest code

**Review by Rajalakshmi-Girish — COMMENTED** [review]
/lgtm

**Pull request merged**